config/schedule.rb

```diff
@@ -50,7 +50,7 @@ def local(time)
 end
 
 every :day, at: local("07:00 am"), roles: [:cron] do
-  rake "appointment_notification:covid_medication_reminders[20000]"
+  AppointmentNotification::ScheduleExperimentReminders.perfom_now
```

Comment:
I don't believe this will work, because this gets written to a plain old cron job via [whenever](https://github.com/javan/whenever).  You need to use `runner` to invoke Ruby like this.

Reply:
I'll make that change.